tests/charts/test_basic_helm_chart.py

```diff
@@ -327,7 +327,8 @@ def test_unsupported_executor(self):
                 },
             )
         assert (
-            'executor must be one of the following: "LocalExecutor", "CeleryExecutor", '
+            'executor must be one of the following: "LocalExecutor", '
```

Comment:
Assert on the error message maybe is not a good idea

Reply:
It's a little fragile, yeah, but honestly it changes so infrequently I think it's okay.